### ISSUE TITLE
locks up!

### ISSUE THREAD

**Issue by dang102** [oxen-io/session-desktop#1268]

**Comment by dang102** [comment]
mint linux 20 CPU usage seems better but it slowly locks up where nothing runs!

**Comment by dang102** [comment]
once I got it to load and not lock up it won't connect to the server!

**Comment by Mikunj** [comment]
@dang102 Is this still occurring?

**Comment by dang102** [comment]
Yes but only in mint 20! and 1.1.2 is there a newer one if so I cant find it?




Sent from Outlook<http://aka.ms/weboutlook>

________________________________

**Comment by Mikunj** [comment]
1.2.1 is a newer version. You can find it here: https://github.com/loki-project/session-desktop/releases/tag/v1.2.1

**Comment by dang102** [comment]
Not sure if it works or not as no one seem to be replying U know that there is no globl looking icon on the left side as everone keeps telling me to click!



Sent from Outlook<http://aka.ms/weboutlook>

________________________________

**Comment by Mikunj** [comment]
> Not sure if it works or not as no one seem to be replying U know that there is no globl looking icon on the left side as everone keeps telling me to click! 

Can you provide more information about "lock up"? What exactly happens when you "lock up" and what situation does it happen in? We can help you if you provide us information + logs (Menu > Debug logs) + screenshots

**Comment by dang102** [comment]
Ok I was running 1.0.9 I think in mint 19.3 working great then I upgraded to mint 20. I load sessions the 1.0.9 worked great! Then I upgraded to 1.1.2 I think and it ran fine the first time loaded the second time just locked up so hard noting worked! Ok then I rebooted tried 1.1.2 again from that time forward it never worked again then you sent me and email saying 1.2.1 was out loaded that would load and looked like it was gona work but couldn't fined the server! then I tried loading on a usb copy of mint 20 no go it would load but would not connect to server. Ok many Linux ppl in the mint    group im in said try backing up to 1.0.9 no go try many other older sessions  some loaded but never connected to a server again,  So then I started getting errors so I formatted the mint partitions and reloaded mint 20 and sessions 1.2.1 no go really look to load and run just failed to send anything and the can't connect to server would pop up once in a while! I also had high and claiming memory use till would get out of memory errors that was in mint So session work everywhere but mint 20 ... So, as I can't get everything working in mint 20 I went back to windows where everything works. Seems like memory leaks maybe mint 20 as it so new! Maybe session 1,2,1 leaking not sure what's up! most lock ups had to have a power off to get out of the lockup! I'm sorry if this is a little out of order and type bad. as one of my brothers has died and the other is in the hospital almost died .. so this is a little hard while crying.. But i didn't want to leave u hanging!


Sent from Outlook<http://aka.ms/weboutlook>

________________________________
